scripts/extractCodeSnippets.js

```diff
@@ -4,6 +4,7 @@ const snippetsFolder = path.join(process.cwd(), './codeSnippets');
 const jsDoc = require(snippetsFolder + '/docs.json');
 const _ = require('lodash');
 const fs = require('fs');
+const isBeta = !!~process.argv.indexOf('--beta');
```

Comment:
I would rather have something like `--package-name "@cognite/sdk-beta"`

Reply:
then we could reuse it for other packages

Reply:
done